py/server/deephaven/table.py

```diff
@@ -1244,15 +1244,22 @@ def format_row_where(self, cond: str, formula: str) -> Table:
             raise DHError(e, "failed to color format rows conditionally.") from e
 
     def layout_hints(self, front: Union[str, List[str]] = None, back: Union[str, List[str]] = None,
-                     freeze: Union[str, List[str]] = None, hide: Union[str, List[str]] = None) -> Table:
+                     freeze: Union[str, List[str]] = None, hide: Union[str, List[str]] = None,
+                     groups: List[dict] = None) -> Table:
         """ Sets layout hints on the Table
 
         Args:
-            front (Union[str, List[str]]): the columns to show at the front
-            back (Union[str, List[str]]): the columns to show at the back
+            front (Union[str, List[str]]): the columns to show at the front.
+            back (Union[str, List[str]]): the columns to show at the back.
             freeze (Union[str, List[str]]): the columns to freeze to the front.
                 These will not be affected by horizontal scrolling.
-            hide (Union[str, List[str]]): the columns to hide
+            hide (Union[str, List[str]]): the columns to hide.
+            groups (List[ColumnGroup]): A list of dicts specifying which columns should be grouped in the UI
+                The dicts can specify the following:
+
+                name (str): The group name
+                children (List[str]): The
+                color (Optional[Union[str, Color]]): The color name, hex string, or color object from deephaven.plot
 
```

Comment:
Which definition does ```Color``` reference to? I don't see the deephaven.plot.color.Color being imported nor tested.

Reply:
Removed `Color` and just used `str` for now. `Color` would have been used by `from python.plot import Colors` and then using `Colors.RED` as an example. The string param can just be the name and the Java method will get the matching color constant